llvm/tools/llvm-shlib/CMakeLists.txt

```diff
@@ -41,9 +41,22 @@ if(LLVM_BUILD_LLVM_DYLIB)
      OR ("${CMAKE_SYSTEM_NAME}" STREQUAL "DragonFly")
      OR ("${CMAKE_SYSTEM_NAME}" STREQUAL "Android")
      OR ("${CMAKE_SYSTEM_NAME}" STREQUAL "SunOS")) # FIXME: It should be "GNU ld for elf"
-    configure_file(
-    ${CMAKE_CURRENT_SOURCE_DIR}/simple_version_script.map.in
-    ${LLVM_LIBRARY_DIR}/tools/llvm-shlib/simple_version_script.map)
+
+    if(NOT "${CMAKE_CFG_INTDIR}" STREQUAL ".")
```

Comment:
Generally we prefer
```
if (...) {
} else {
}
```
over
```
if (!...) {
} else {
}
```

CMake is similar.

Reply:
The "if" style is adjusted in [72e1dc0](https://github.com/llvm/llvm-project/pull/65451/commits/72e1dc0bc7768a131df4b1dcb5d6bda44afc4bba).